README.md

```diff
@@ -853,6 +854,75 @@ var timestamp = time.ZonedDateTime.now().plusMinutes(5);
 console.log(timestamp.getMillisFromNow());
 ```
 
+### Quantity
+
+The `Quantity` class greatly simplifies Units of Measurement (UoM)/Quantity handling by providing unit conversion, comparisons and mathematical operations.
+
```

Comment:
I would define what is meant by a Quantity ("the measurement and the unit of measurement) and the unit itself.

Reply:
Maybe provide a link to the UoM page where you say "Units of Measurement (UoM)". I see you linked to it below, but I like the idea of moving that link up to here.